src/Lykke.Service.PayInternal/Controllers/TransactionsController.cs

```diff
@@ -117,10 +118,7 @@ public class TransactionsController : Controller
                 var transactions = await _transactionsService.GetTransactionsByPaymentRequestAsync(paymentRequestId);
```

Comment:
If you return empty collection instead of null the code will look much more readable.